queries.sql

```diff
@@ -7,4 +7,49 @@ select name, date_of_birth from animals WHERE name NOT IN ('Agumon','Pikachu');
 select name, escape_attempts, weight_kg from animals WHERE weight_kg > 10.5;
 select * from animals WHERE neutered = 'true';
 select * from animals WHERE name NOT IN ('Gabumon');
-select * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+select * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- UPDATE queries OF PART 2
+
+update animals set species = 'unspecified';
+ROLLBACK;
+select * from animals;
+
+Update animals set species = 'digimon' WHERE name like '%mon';
+select * from animals;
+
+Update animals set species = 'pokemon' WHERE species isnull;
+select * from animals;
+
+COMMIT;
+
+select * from animals;
+
+drop table animals;
+Rollback;
```

Comment:
- [OPTIONAL] Regarding this queries: `Inside a transaction delete all records in the animal's table, then roll back the transaction.`  you may also make it in this may 👍🏻 

```sql
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```